pkg/templates/charts/toggle/console-mce/templates/console-prometheus-rules.yaml

```
@@ -0,0 +1,12 @@
+# Copyright Contributors to the Open Cluster Management project
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: acm-console-prometheus-rules
+  namespace: openshift-monitoring
```

We are no longer deploying `PrometheusRules` or `ServiceMonitors` in the `openshift-monitoring` namespace, if the resource is being manged by ACM.

```suggestion
  namespace: {{ .Values.global.namespace }}
```